scripts/junit_report.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable no-console */
+
+const semver = require('semver')
+const { execSync } = require('child_process')
+
+function uploadJUnitXMLReport () {
+  if (semver.lt(process.version, '10.24.1')) {
+    console.log('Node version incompatible with @datadog/datadog-ci. Skipping step.')
+    return
+  }
+  if (process.env.CIRCLE_PR_NUMBER) {
+    console.log('Running in a fork. Skipping step.')
+    return
+  }
+  // we install @datadog/datadog-ci
+  execSync('yarn add --dev @datadog/datadog-ci@0.13.1', { stdio: 'inherit' })
```

Comment:
Any way this can be added as a dev dependency even if it doesn't support older Node versions? I feel like we need a better solution for this type of dependency since it happens very often that there are compatibility issues.

Reply:
I don't think I've ever seen that. Is there a mechanism for that?

Reply:
that's a good point. I'll add `--no-save` to `npm install` (yarn does not seem to support it)

Reply:
Mixing `yarn` and `npm` is not supported by either and is not something we should be doing. It looks like `yarn` left this functionality explicitly to avoid corruption between the info in `yarn.lock` and `node_modules`, which makes sense and is in line with my first comment above that dependencies should be explicit.

Can `datadog-ci` be installed globally instead maybe?

Reply:
yeah we can do it globally 👍

Reply:
done